components/studio/habitat/plan.sh

```diff
@@ -46,21 +45,3 @@ do_install() {
     "$pkg_prefix"/libexec/"$hab_dir"/bin/hab
   ln -sv "$hab_dir"/bin/hab "$pkg_prefix"/libexec/hab
 }
-
-# Turn the remaining default phases into no-ops
```

Comment:
It's not clear to me why this is no longer needed. Don't we have default implementations of these actions? If they're not overridden, they won't be no-ops, right?

Reply:
That's a great thing to call out.  It used to be the case that `pkg_source` was "required" ( this was  two years ago so my memory is a bit fuzzy on what required actually meant ), which meant having to override the download/unpack/verify steps as there was nothing to actually do. 

Now you are able to omit the `pkg_source` and plan-build will automatically skip over those steps so we no longer need to no-op them.   

I'm not entirely sure on the `do_prepare` no-op, that may be an even older leftover from early iterations on plan-build, but the default for it is currently a no-op so we can safely omit it as well.